js/src/common/helpers/highlight.ts

```diff
@@ -4,21 +4,20 @@ import { truncate } from '../utils/string';
  * The `highlight` helper searches for a word phrase in a string, and wraps
  * matches with the <mark> tag.
  *
- * @param {String} string The string to highlight.
- * @param {String|RegExp} phrase The word or words to highlight.
- * @param {Integer} [length] The number of characters to truncate the string to.
+ * @param string The string to highlight.
+ * @param phrase The word or words to highlight.
+ * @param [length] The number of characters to truncate the string to.
  *     The string will be truncated surrounding the first match.
- * @return {Object}
  */
-export default function highlight(string, phrase, length) {
+export default function highlight(string: string, phrase: string | RegExp, length: number): Object | string {
```

Comment:
The return type should be set to `Mithril.Vnode` here

Reply:
fixed